### ISSUE TITLE
Health regeneration in gang zones

### ISSUE THREAD

**Issue by SophiaLVP** [LVPlayground/playground#641]
So recently a proper gang zone and gang attributes were published and those look great. However we still need couple of tweaks to make things even a lot better so here we go

1. The first and best attribute could be the the health regeneration when player is standing in the zone and his/her health is less than 35. Health should regenerate 1 hp per 10-15 seconds and that depends on the zone infrastructure. Although I vote for 15 seconds personally as it brings balance in game as well mostly during gang wars around territories.

2. Allow an ability for leaders to set withdraw amount for gang bank per day. This means when player reach certain withdraw limit set by their leaders, he/she couldn't be able to withdraw anymore for the rest of the day. Although there's no need for gang banking limit as "the more the merrier" plays it's part.

3. Allow a cool down time for gang withdraw command. I have seen so much spam today and it's kind of annoying. Cool down time for withdrawing gang cash is also in the hands of leaders so it will be up to them to decide the cool time. I am suggesting this because we are not considering ourselves (alone) here. We are talking about a whole gang and anyone can withdraw gang cash anytime and many times as they like until they hit the certain limit as suggested above.

Ideas welcome!

**Comment by RussellLVP** [comment]
Thank you for the suggestions!

  1. I like the idea to be able to purchase health regeneration. I think it should be a bit faster though: it takes 16.5 minutes with your proposed speed to get from 1 HP to 100 HP. (Ignoring the ceiling at 35 for now.)
  2. I've considered this, but it's not entirely straightforward to implement. Let me think about this some more.
  3. I don't quite understand what you mean. A _cool down_ as in allowing people to only withdraw money once per minute? Or, in relation to suggestion 2, a cool down regarding a daily limit?

**Comment by SophiaLVP** [comment]
>A cool down as in allowing people to only withdraw money once per minute?

Exactly. Imagine everyone withdrawing $1 each second from gang bank. It shows you a notification about it. Now imagine 4 gang members doing the same thing together. It's a hell of a spam. I just interconnected it with suggestion number 2

>I like the idea to be able to purchase health regeneration. I think it should be a bit faster though: it takes 16.5 minutes with your proposed speed to get from 1 HP to 100 HP. (Ignoring the ceiling at 35 for now.

I would like to suggest renting instead of purchasing just like we rent gang chat encryption on a monthly basis.

**Comment by Specifer** [comment]
Everything is in hands of the leader? Nah, Leaders and managers should be given same perks so there should be a limit where you can't go lesser than 1 million (maybe) . And this also goes to the members they can't withdraw less that 1 million. And for the cool down too, No. I suggest the regeneration time, like if some or all gang members withdraw upto 25 million(more) . They have to spend some time of 48hours online to be able to withdraw again. Cool down for it is not good since its not Good xd. BTW I have some things to add in achievements which can be for gangs. Like if you bank your money to gang account you will finish several achievements depending on the amount you bank.

**Comment by SophiaLVP** [comment]
No, your suggestion will make things worse than we have now. I've suggested mine on some reasoning and observation.

**Comment by RussellLVP** [comment]
Based on feedback from quite a few gangs, suggestions (2) and (3) are not necessary right now. Gangs have all the tools to moderate this themselves. Let's rescope this to health regeneration within their zone.
